chalice/pipeline.py

```diff
@@ -9,18 +9,36 @@
 def create_pipeline_template(config):
     # type: (Config) -> Dict[str, Any]
     pipeline = CreatePipelineTemplate()
-    return pipeline.create_template(config.app_name)
+    return pipeline.create_template(config.app_name,
+                                    config.lambda_python_version)
+
+
+class InvalidCodeBuildPythonVersion(Exception):
+    def __init__(self, version):
+        # type: (str) -> None
+        super(InvalidCodeBuildPythonVersion, self).__init__(
+            'CodeBuild does not yet support python version %s.' % version
+        )
 
 
 class CreatePipelineTemplate(object):
 
+    _CODEBUILD_IMAGE = {
+        'python2.7': 'python:2.7.12',
+    }
+
     _BASE_TEMPLATE = {
         "AWSTemplateFormatVersion": "2010-09-09",
         "Parameters": {
             "ApplicationName": {
                 "Default": "ChaliceApp",
                 "Type": "String",
                 "Description": "Enter the name of your application"
+            },
+            "CodeBuildImage": {
```

Comment:
Well I was just thinking even if they're deploying the cfn template manually, it's still something they can parameterize themselves (say using `aws cloudformation deploy`).  I wouldn't think we would need to add ` --codebuild-image` ourselves.